src/StatusCodeError.js

```diff
@@ -28,7 +28,9 @@ class StatusCodeError extends Error {
       statusCode = 504;
     }
     if (err instanceof FetchError) {
-      if (err.code === 'ECONNRESET' || err.code === 'ETIMEDOUT') {
+      if (err.code === 'ECONNRESET' || err.code === 'ECONNREFUSED') {
+        statusCode = 503;
```

Comment:
Are you sure? AFAIR we've discussed this in the past and decided to propagate `ECONNRESET` as 504 as it's a temporary issue. That's the case in `helix-static` and `helix-content-proxy`.

https://github.com/adobe/helix-static/pull/518
https://github.com/adobe/helix-content-proxy/pull/319

Reply:
hmm.. and `ECONNREFUSED` ?

Reply:
ok. let's keep it temporary... I revert this change